site/styles/main.scss

```diff
@@ -349,3 +349,24 @@ s72-classification-label {
 h1, h2, h3, h4, h5, h6, p, .title {
   word-wrap: break-word;
 }
+
+p, .s72-prompt, .s72-searchresults-empty-content, span, .s72-modal-body .s72-shopping-done {
```

Comment:
One per line please!  Putting loads of stuff on one line makes future diff-checking a pain in the butt!

Reply:
Shifted most of it in the relevant files. Some of them don't have a separate file, eg: comes under span or a div